http/v1.md

```diff
@@ -0,0 +1,488 @@
+
+## REST API
+
+### GET /v1/asset_pairs
+
+Retrieves a list of available asset pairs and the information required to trade them. This endpoint should be [paginated](#pagination).
+
+#### Parameters
+
+*   assetDataA=&assetDataB [string]: these are assetData fields. Returns asset pairs that contain assetDataA and assetDataB (in any order). Setting only assetDataA or assetDataB returns pairs filtered by that asset only (optional)
+
+#### Response
+
+[See response schema](https://github.com/0xProject/0x-monorepo/blob/development/packages/json-schemas/schemas/relayer_api_token_pairs_response_schema.ts#L1)
+
+```
+[
+    "total": 43,
+    "page": 1,
+    "per_page": 100,
+    "records": [
+        {
+            "assetDataA": {
+                "minAmount": "0",
+                "maxAmount": "10000000000000000000",
+                "precision": 5
+            },
+            "assetDataB": {
+                "minAmount": "0",
+                "maxAmount": "50000000000000000000",
+                "precision": 5
+            }
+        },
+        ...
+    ]
+]
+```
+
+*   `minAmount` - the minimum trade amount the relayer will accept
+*   `maxAmount` - the maximum trade amount the relayer will accept
+*   `precision` - the desired price precision a relayer would like to support within their orderbook
+
+### GET /v1/orders
+
+Retrieves a list of orders given query parameters. This endpoint should be [paginated](#pagination). For querying an entire orderbook snapshot, the [orderbook endpoint](#get-v1orderbook) is recommended.
+
+#### Parameters
+
+Filter parameters:
+  *   makerAssetType [string]: returns orders where the maker asset is of certain asset type (example: `ERC20`, `ERC721`)
+  *   takerAssetType [string]:  returns orders where the taker asset is of certain asset type (example: `ERC20`, `ERC721`)
+  *   makerAssetAddress [string] returns orders where the contract address for the maker asset matches the value specified
+  *   takerAssetAddress [string] returns orders where the contract address for the taker asset matches the value specified
+
+Order specific parameters:
+  *   exchangeAddress [string]: returns orders created for this exchange address
+  *   senderAddress [string]: returns orders with the specified senderAddress
+  *   makerAssetData [string]: returns orders with the specified makerAssetData
+  *   takerAssetData [string]: returns orders with the specified takerAssetData
+  *   traderAssetData [string]: returns orders where either makerAssetData or takerAssetData has the value specified
+  *   makerAddress [string]: returns orders with the specified makerAddress
+  *   takerAddress [string]: returns orders with the specified takerAddress
+  *   traderAddress [string]: returns orders where either makerAddress or takerAddress has the value specified
+  *   feeRecipientAddress [string]: returns orders where feeRecipientAddress is feeRecipient address
+
+All parameters are optional.
```

Comment:
Do you mean supporting them is optional?

Reply:
I interpreted that to mean that sending them in an API call is optional.

Reply:
^ This seems to be a recurring miscommunication :P 

Is there a clearer way to distinguish between optional to implement vs. optional to include in the API call?

Reply:
Let's just say: `All parameters are optional for the caller to supply.`